platformio.ini

```diff
@@ -0,0 +1,17 @@
+;PlatformIO Project Configuration File
+;
+;   Build options: build flags, source filter
+;   Upload options: custom upload port, speed and extra flags
+;   Library options: dependencies, extra library storages
+;   Advanced options: extra scripting
+;
+; Please visit documentation for the other options and examples
+; https://docs.platformio.org/page/projectconf.html
+[env]
+platform = espressif32
+board = pico32
+framework = espidf
+upload_port = /dev/ttyUSB*
```

Comment:
why hardcode the upload_port if it can be specified by an pio run argument?

Reply:
If several ttyUSB (esp. ESP32 of the same type) devices are connected the individual user can select which ttyUSB to use. So i would prefer this version over an argument in a dockerfile

Reply:
the device can be selected via make like `make pio-flash DEV=/dev/ttyUSB1` without touching any files. the Makefile could be extended by a device check if the fallback doesn't exist, or even choose a device

Reply:
will we keep the `upload_port`? Port is now managed by the makefile